packages/frontend/src/hooks/useQueryResults.tsx

```diff
@@ -163,8 +167,17 @@ export const useUnderlyingDataResults = (
 };
 
 // This hook will be used for getting charts in view mode and dashboard tiles
-export const useChartResults = (chartUuid: string, filters?: Filters) => {
-    const queryKey = ['savedChartResults', chartUuid, JSON.stringify(filters)];
+export const useChartResults = (
+    chartUuid: string,
+    filters?: Filters,
+    invalidateCache?: boolean,
+) => {
+    const queryKey = [
+        'savedChartResults',
+        chartUuid,
+        JSON.stringify(filters),
+        invalidateCache,
```

Comment:
@magnew do we actually need to track the cache in the state? I don't think we need the charts to rerun when this status changes. We just need the charts to re-run when the user hits refresh

Reply:
Right, this is the part where we are fighting with react query a bit. When you hit refresh, we need to invalidate the queries. One way to do that is to put some cache indicator in the query key. That's why I'm keeping it here, but I would like to isolate it from the view code.